src/components/Loading/Loading.tsx

```diff
@@ -0,0 +1,11 @@
+import styles from './Loading.module.scss';
+import logo from '../../assets/Temp_Roadwatch_logo.svg';
+
+export function Loading(): JSX.Element {
```

Comment:
I recommend exporting this as `default` and then modifying the `index.ts` file accordingly.
```suggestion
export default function Loading(): JSX.Element {
```